components/automate-ui/src/app/modules/infra-proxy/reset-client-key/reset-client-key.component.scss

```diff
@@ -0,0 +1,255 @@
+
+    .input-margin {
+      margin-bottom: 30px;
+    }
+
+    .id-margin {
+      width: 100%;
+    }
+
+    .left-box {
+      float: left;
+      height: auto;
+      width: 40%;
+    }
+
+    .right-box {
+      float: left;
+      height: auto;
+      width: 60%;
+    }
+
+    .title-text {
+      font-size: 36px;
+      font-weight: 900;
+      margin-top: 72px;
+      margin-bottom: 22px;
+      line-height: 1;
+    }
+
+    .action-text {
+      font-size: 24px;
+      font-weight: 900;
+      line-height: 28px;
+    }
+
+    .left-align {
+      line-height: 30px;
+      margin-bottom: 63px;
+      margin-left: 43px;
+      text-align: left;
+    }
+
+    .big-icon {
+      z-index: 999999;
+      position: relative;
+    }
+
+    .key-tab-body {
+      display: flex;
+      background: $chef-white;
+      border: 0.5px solid $chef-key-tab;
+      height: 300px;
+
+      chef-snippet {
+        height: 298px;
+        width: 99.9%;
+      }
+
+    }
+
+    .detail {
+      background: $chef-success;
+      color: $white;
+      font-size: 12px;
+      line-height: 20px;
+      padding: 9px;
+      margin-bottom: 32px;
+
+      .icon-wrap {
+        display: inline-block;
+        position: relative;
+        width: 40px;
+
+        i {
+          display: block;
+          height: 16px;
+          width: 16px;
+          background-image: url('/assets/img/icon-check-circle.png');
+          position: absolute;
+          top: -30px;
+          left: 8px;
+        }
+
+      }
+
+      p {
+        display: inline-block;
+        font-size: 12px;
+        line-height: 20px;
+        letter-spacing: 0.25px;
+        width: 80%;
+        color: $white;
+        text-transform: capitalize;
+        margin-bottom: 0;
+      }
+
+    }
+
+    .button-bar {
+      text-align: right;
+      padding-top: 24px;
+
+      chef-button {
+        margin-bottom: 0;
+      }
+
+      #reset-button-object-modal, #download-button-object-modal {
+        margin-right: 0;
+      }
+
+      chef-loading-spinner {
+        position: relative;
+        top: 2px;
+        margin-right: 5px;
+      }
+    }
+  }
+
+  ::ng-deep {
+    .modal {
+      box-shadow: 0px 0px 30px 0px #CCC;
+      outline: none;
+      width: 700px;
+
+      .options {
+        float: right;
+
+        chef-button[primary] {
+          background-color: $chef-critical;
+          border-color: $chef-critical;
+
+          &:hover, &:focus {
+            background-color: $chef-critical-dark;
+            border-color: $chef-critical-dark;
+          }
+
+          chef-loading-spinner {
+            color: $chef-critical-light;
+          }
+
+        }
+
+        chef-button[tertiary] {
+          color: $chef-critical;
+
+          &:hover, &:focus {
+            background-color: $chef-critical-light;
+            border-color: $chef-critical-light;
+          }
+        }
+
+      }
+
+      chef-button {
+
+        &:focus {
+          outline: 0.5px solid $chef-critical-dark;
```

Comment:
I'm noticing that you are using 0.5px for the outlines, is this from the new UX mockups?  Currently across our site, all the focus rings are 1px.

I'm asking because I know much of the UX is getting updated, so I want to make sure i'm not missing anything, as 0.5px is used quite a bit in this PR

Reply:
@SEAjamieD , Yeah it's new design for focus part, UX team suggestions we added 0.5px.

Reply:
thank you!